app/controllers/application_controller.rb

```diff
@@ -13,4 +14,19 @@ def authenticate
       password: ENV['BASIC_AUTH_LOGIN_PASSWORD']
     )
   end
+
+  def use_locale(&action)
+    locale = locale_params[:locale] || I18n.default_locale
```

Comment:
Just one thing as I'm curious: How does the rails understand `locale_params[:locale]`? I'm used to set that in the browser, based on `navigator.language` among other things. Does Rails do some magic with `Accept-Language` header? Or do you send the locale in the query string: `?locale=fi`?

Reply:
For this use case, I opted to set them using a query param `locale=fi`, yes. This makes it easy to send the link to the Finnish translation to Finnish participants and the link for German translation to German participants. In this use case, I think that’s simply the most reliable and easy way, although for most other use cases, using the `Accept-Language` header would probably be better.